docs/en/transform-v2/tikadocument.md

```diff
@@ -0,0 +1,307 @@
+# TikaDocument
+
+> TikaDocument Transform Plugin
+
+## Description
+
+The `TikaDocument` transform plugin uses [Apache Tika](https://tika.apache.org/) to extract text content and metadata from various document formats including PDF, Microsoft Office documents (Word, Excel, PowerPoint), plain text, HTML, XML, and many other file formats. This transform converts binary document data into structured text content and metadata fields.
```

Comment:
> This transform converts binary document

This part is easily overlooked by users. I think we can open a separate chapter to introduce which source configurations can use this transform.